pkgs/by-name/un/undetected-chromedriver/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+
+  chromedriver,
+  python3,
+
+  testers,
+  undetected-chromedriver,
+}:
+let
+  specSystem =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      "linux64"
+    else if stdenv.hostPlatform.system == "x86_64-darwin" then
+      "mac-x64"
+    else if stdenv.hostPlatform.system == "aarch64-darwin" then
+      "mac-arm64"
+    else
+      throw "Unsupported system";
+in
+chromedriver.overrideAttrs (old: {
+  pname = "undetected-chromedriver";
+
+  nativeBuildInputs = old.nativeBuildInputs ++ [
+    (python3.withPackages (ps: [ ps.undetected-chromedriver ]))
+  ];
+
+  postPatch = ''
+    export HOME=$(mktemp -d)
```

Comment:
Not a big fan of this entire thing, it's probably a better idea to just copy + patch the executable.

Reply:
Tried it, seems to work fine: https://github.com/K900/nixpkgs/commit/cee4c004cd8cb46b5a5cbddf2735d6d008453b25

Reply:
(also that requires https://github.com/NixOS/nixpkgs/commit/174cc2e35c951b4f31449ca9adb4b91561611891)

Reply:
Right, with the build from the source this idea is indeed better. Before your changes, the `${chromedriver}/bin/chromedriver` file was a shell wrapper around a built binary, which required to hack around the dependencies. Seemed like overriding the build process was easier in this respect.

Reply:
Yeah, I got that after looking at the binary expression.